pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <groupId>it.gov.pagopa.api-config</groupId>
   <artifactId>starter</artifactId>
-  <version>1.8.0</version>
+  <version>1.9.0</version>
```

Comment:
why the manual versioning?

Reply:
I didn't make the change, I think it's automatic